app/util/sentry/utils.js

```diff
@@ -6,6 +6,239 @@ import DefaultPreference from 'react-native-default-preference';
 import { regex } from '../regex';
 import { AGREED, METRICS_OPT_IN } from '../../constants/storage';
 import { isTest } from '../test/utils';
+import { store } from '../../store';
+
+/**
+ * This symbol matches all object properties when used in a mask
+ */
+const AllProperties = Symbol('*');
+
+// This describes the subset of background controller state attached to errors
+// sent to Sentry These properties have some potential to be useful for
+// debugging, and they do not contain any identifiable information.
+export const SENTRY_STATE = {
```

Comment:
S: Now that I read this a bit more I feel it could be called `sentryStateMask` for more clarity (and also avoid uppercase const names, that's not in guidelines and usually more for env vars.)
```suggestion
export const sentryStateMask = {
```